app/assets/stylesheets/navigation.scss

```diff
@@ -6,53 +6,67 @@ nav {
 
   background-color: $bg-color;
 
-  .button-collapse {
-    i {
-      color: $font-color;
-      font-size: 1.8rem;
-      margin-right: 1rem;
-    }
+  .container {
+    width: 100%;
+    margin: 0 auto 0 15px;
   }
 
-  .brand-logo {
+  a.brand-logo {
+    padding: 0 15px;
     color: $font-color;
     font-size: 1.2rem;
     font-weight: 700;
   }
+}
 
-  ul.right {
-    i { color: lighten($secondary-color, 25%); }
-  }
+a.button-collapse.top-nav {
+  position: absolute;
```

Comment:
Properties should be ordered color, float, font-size, left, margin-left, position, text-align, top, z-index